modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java

```diff
@@ -22,6 +27,9 @@ public class CredentialCenter {
 
     private final OrchestratorService orchestratorService;
 
+    private final Map<Csp, Map<String, AbstractCredentialInfo>> credentialInfoMap =
+            new Hashtable<>();
```

Comment:
Suggest using  CurrentHasMap instead of HashTable.

Reply:
ok